workflow/tests/base_results/results.csv

```diff
@@ -316,7 +316,7 @@ base-schedules-detailed-smooth.xml,34.99,34.99,14.52,0.0,0.0,0.0,0.0,0.0,0.0,0.3
 base-schedules-simple.xml,35.0,35.0,14.62,0.0,0.0,0.0,0.0,0.0,0.0,0.38,3.51,0.68,9.16,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.11,8.37,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.62,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,13.83,11.41,9.23,0.61,0.0,0.0,0.0,0.0,2025.0,3456.0,20.49,20.7,0.0,3.88,3.97,0.54,3.98,0.71,-0.1,0.0,0.0,3.05,-0.11,5.01,0.0,0.76,0.0,3.35,-11.32,0.0,-0.03,-0.46,-0.01,3.85,-0.03,8.93,0.0,0.0,-5.99,-0.07,-1.04,-6.56,-0.15,0.0,3.03,9.96,1355.0,998.0,11400.0,2616.0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
-base-simcontrol-calendar-year-custom.xml,35.01,35.01,14.5,0.0,0.0,0.0,0.0,0.0,0.0,0.38,3.51,0.68,9.16,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.12,8.38,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.5,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,13.71,11.43,9.23,0.61,0.0,0.0,0.0,0.0,2140.0,3445.0,20.48,20.68,0.0,3.89,3.98,0.54,4.02,0.7,-0.17,0.0,0.0,3.05,-0.07,5.01,0.0,0.76,0.0,3.33,-11.47,0.0,-0.04,-0.46,-0.02,3.88,-0.03,9.11,0.0,0.0,-6.01,-0.02,-1.05,-6.63,-0.15,0.0,3.03,9.82,1355.0,998.0,11400.0,2616.0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+base-simcontrol-calendar-year-custom.xml,34.98,34.98,14.53,0.0,0.0,0.0,0.0,0.0,0.0,0.38,3.49,0.67,9.16,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.11,8.37,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.53,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,13.73,11.33,9.23,0.61,0.0,0.0,0.0,0.0,2140.0,3451.0,20.46,20.7,0.0,3.89,3.98,0.54,4.0,0.71,-0.09,0.0,0.0,3.07,-0.11,5.01,0.0,0.76,0.0,3.33,-11.46,0.0,-0.02,-0.45,-0.01,3.86,-0.02,8.93,0.0,0.0,-5.98,-0.07,-1.04,-6.54,-0.15,0.0,3.01,9.83,1355.0,998.0,11400.0,2616.0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
```

Comment:
I assume these small diffs are due to the Day of Week for Start Day in the simulation changing?

Reply:
Yes I believe so.